.gitignore

```diff
@@ -1 +1,2 @@
 target/
+.idea/
```

Comment:
You should set up global excludes on your machine and remove this IDE-specific ignore from the PR. See http://stackoverflow.com/questions/7335420/global-git-ignore